prql-compiler/src/sql/gen_expr.rs

```diff
@@ -400,13 +400,14 @@ pub(super) fn translate_query_sstring(
 ) -> Result<sql_ast::Query> {
     let string = translate_sstring(items, context)?;
 
+    let re = Regex::new(r"(?i)^SELECT(\s|\n|\r)").unwrap();
```

Comment:
FYI this can be `r"(?i)^SELECT\b"`

Reply:
Thanks for pointing that, I didn't know `\b`...
I will open another PR.